tests/30rooms/60anonymousaccess.pl

```diff
@@ -687,6 +687,87 @@ sub check_events
       });
    };
 
+test "GET /publicRooms includes avatar URLs",
+   requires => [ $main::API_CLIENTS[0], local_user_fixture() ],
+
+   check => sub {
+      my ( $http, $user ) = @_;
+
+      Future->needs_all(
+         matrix_create_room( $user,
+            visibility => "public",
+            room_alias_name => "nonworldreadable",
```

Comment:
There's a slight danger of these "unique" alias names becoming fragile across many test files.
I don't have any better suggestion currently, but perhaps something to think about for the future. Maybe a fixture for allocating new unique alias names?